aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -96,6 +98,27 @@ class AwsFluentClientDecorator : ClientCodegenDecorator {
             }
         }
     }
+
+    override fun protocolTestGenerator(
+        codegenContext: ClientCodegenContext,
+        baseGenerator: ProtocolTestGenerator,
+    ): ProtocolTestGenerator = DefaultProtocolTestGenerator(
+        codegenContext,
+        baseGenerator.protocolSupport,
+        baseGenerator.operationShape,
+        renderClientCreation = { params ->
+            rustTemplate(
+                """
+                let config = ${params.configBuilderName}
+                    .http_connector(${params.connectorName})
+                    .region(crate::config::Region::new("us-east-1"))
```

Comment:
another way to do this is to use vendor params and customize the tests themselves...I wonder if that's better actually.

https://github.com/awslabs/smithy-rs/blob/6beec943ab767a791a414bbaef1acb4b363f7042/aws/sdk/aws-models/glacier-tests.smithy#L23-L29

I think the existing tests almost all already have a region set?

Reply:
Oh, I didn't know this was available. Easy fix.